src/App.js

```diff
@@ -1,23 +1,29 @@
-import logo from './logo.svg';
+
 import './App.css';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import ListBeers from "./pages/ListBeers"
+import Home from "./pages/Home"
+import RandomBeer from "./pages/RandomBeer"
+import NewBeer from './pages/NewBeer';
+import SingleBeer from './pages/SingleBeer';
+
 
 function App() {
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/listbeers" element={<ListBeers />} />
+        <Route path="/newbeer" element={<NewBeer />} />
+        <Route path="/randombeer" element={<RandomBeer />} />
+        <Route path="/singlebeer" element={<SingleBeer />} />
```

Comment:
Remember that the path should be to that single beer you're passing. So you should use what you passed as the `params` on the `SingleBeer` page, `beerId`.

`<Route path="/:beerId" element={<SingleBeer />} />`